packages/fluentui/react-northstar/src/types.ts

```diff
@@ -159,4 +159,5 @@ export interface ProviderContextPrepared extends StylesContextValue {
   // `target` can be undefined for SSR
   target: Document | undefined;
   telemetry: Telemetry | undefined;
+  rtl: any;
```

Comment:
Why was this added? `rtl` already exists in `StylesContextValue`

Reply:
this was high probably added by mistake, or automatically by editor, removing and I cannot see any typing complains :)